docs/api/shallow.md

```diff
@@ -48,7 +48,7 @@ describe('<MyComponent />', () => {
   - `options.context`: (`Object` [optional]): Context to be passed into the component
   - `options.disableLifecycleMethods`: (`Boolean` [optional]): If set to true, `componentDidMount`
 is not called on the component, and `componentDidUpdate` is not called after
-[`setProps`](ShallowWrapper/setProps.md) and [`setContext`](ShallowWrapper/setContext.md). Default to `false`.
+[`setProps`](ShallowWrapper/setProps.md) and [`setContext`](ShallowWrapper/setContext.md). Defaults to `true`.
```

Comment:
Am happy to change this to explain that it defaults to `true` for v3 but `false` for v2, if that is more helpful for the readers. I thought I would just start with what is true for `enzyme@latest`, though.